### PR TITLE
BL-2367: Add command to copy books within Bloom

### PR DESCRIPTION
**Pull request by phillip-hopper**

### CODE REVIEW

**Comment by JohnThomson** [comment]
It's very helpful to a reviewer (and possibly later students of the version history) if the commit message at least briefly summarizes other major changes (e.g., this one also adds a triangle with a subset pull-down menu next to the selected thumbnail). Probably not worth another change at this point...just for future reference.

File: src/BloomExe/CollectionTab/LibraryListView.cs [hunk 1]
```diff
@@ -658,9 +704,17 @@ private void HighlightBookButton(BookInfo bookInfo)
 			foreach (var btn in AllBookButtons())
 			{
 				if (btn.Tag == bookInfo)
+				{
 					btn.FlatAppearance.BorderColor = Palette.TextAgainstDarkBackground;
+					_bookTriangle.Left = btn.Left + btn.Width - _bookTriangle.Width - 2;
```
Comment (JohnThomson) [inline src/BloomExe/CollectionTab/LibraryListView.cs]: Is there a better name for this method now it is doing more than highlighting the indicated button? DisplayBookButtonAsActive?

File: src/BloomExe/CollectionTab/LibraryListView.cs [hunk 2]
```diff
@@ -38,6 +39,8 @@ public partial class LibraryListView : UserControl
 		private bool _primaryCollectionReloadPending;
 		private bool _disposed;
 		private BookCollection _downloadedBookCollection;
+		private PictureBox _bookTriangle;
```
Comment (JohnThomson) [inline src/BloomExe/CollectionTab/LibraryListView.cs]: I'm not keen on this name. Took me quite a while to figure out that this was an icon to provide an alternative way to get at (part of) the right-click menu, and that it moves next to the active thumbnail. Would _contextMenuButton be better? Even _thumbnailContextMenuButton? It's not used so much that a long informative name is inappropriate. A comment summarizing what it is for might be helpful, too.

**Comment by JohnThomson** [comment]
Basically good...couple of minor issues..what do you think? Are they worth changing?

File: src/BloomExe/CollectionTab/LibraryListView.cs (hunk 1 above)
Comment (phillip-hopper) [inline src/BloomExe/CollectionTab/LibraryListView.cs]: Updated

File: src/BloomExe/CollectionTab/LibraryListView.cs (hunk 2 above)
Comment (phillip-hopper) [inline src/BloomExe/CollectionTab/LibraryListView.cs]: Renamed.

**Comment by phillip-hopper** [comment]
Commit message updated.

**Pull request merged**